index.md

```diff
@@ -1,5 +1,16 @@
-A group of friends that decided to contribute to the community by starting to organize events where we could gather everyone interested in all the topics regarding testing. 
-Check our events <a href="https://portotestersmeetup.eventbrite.com" target="_blank">here</a>.
+
+# About
+
+<p>
+  A group of friends that decided to contribute to the community by starting to organize events where we could gather everyone interested in all the topics regarding testing.
+</p> 
+<p>
+  At each event we start with one or two presentations and then discuss these and other themes afterwards along with a coffee break provided by the host company.
```

Comment:
Remove afterwards or replace "and then" with "and afterwards"

Reply:
done